scripts/maint/practracker/metrics.py

```diff
@@ -8,6 +8,14 @@ def file_len(f):
         pass
     return i + 1
 
+def get_include_count(f):
+    """Get number of #include statements in the file"""
+    include_count = 0
+    for line in f:
+        if re.match(r' *# *include', line):
```

Comment:
Maybe this should be `\s*` instead of ` *`, since tabs can also appear in this context. On the other hand, Tor doesn't allow tabs, so maybe we don't need to care.

Reply:
Included this improvement in the post-merge ticket #29746, since it's a good-to-have but doesn't actually influence results atm.